overlord/configstate/configcore/corecfg.go

```diff
@@ -120,6 +103,11 @@ func Run(tr Conf) error {
 		return err
 	}
 
+	// Export experimental.* flags to a place easily accessible from C.
```

Comment:
How about `// Export experimental.* flags to a place easily accessible from snapd helpers.` ?